Jenkinsfile

```diff
@@ -4,7 +4,8 @@
  * allowing one to test against multiple Jenkins versions.
  */
 buildPlugin(useContainerAgent: true, configurations: [
-  [ platform: 'linux', jdk: '8' ],
   [ platform: 'linux', jdk: '11' ],
+  [ platform: 'linux', jdk: '17' ],
```

Comment:
doesn't appear to be working:

```
[](https://ci.jenkins.io/blue/organizations/jenkins/Core%2Fwinstone/detail/PR-181/15/pipeline/48#step-55-log-2)[](https://ci.jenkins.io/blue/organizations/jenkins/Core%2Fwinstone/detail/PR-181/15/pipeline/48#step-55-log-3)[](https://ci.jenkins.io/blue/organizations/jenkins/Core%2Fwinstone/detail/PR-181/15/pipeline/48#step-55-log-4)[](https://ci.jenkins.io/blue/organizations/jenkins/Core%2Fwinstone/detail/PR-181/15/pipeline/48#step-55-log-5)[](https://ci.jenkins.io/blue/organizations/jenkins/Core%2Fwinstone/detail/PR-181/15/pipeline/48#step-55-log-6)[2022-02-08T03:41:05.381Z] + mvn --batch-mode --show-version --errors --no-transfer-progress --update-snapshots -Dmaven.repo.local=/home/jenkins/workspace/Core_winstone_PR-181@tmp/m2repo -Dmaven.test.failure.ignore -Dspotbugs.failOnError=false -Dcheckstyle.failOnViolation=false -Dcheckstyle.failsOnError=false -Penable-jacoco -Dset.changelist help:evaluate -Dexpression=changelist -Doutput=/home/jenkins/workspace/Core_winstone_PR-181@tmp/changelist clean install

[2022-02-08T03:41:05.381Z] Apache Maven 3.8.4 (9b656c72d54e5bacbed989b64718c159fe39b537)

[2022-02-08T03:41:05.381Z] Maven home: /usr/share/maven

[2022-02-08T03:41:05.381Z] Java version: 11.0.13, vendor: Eclipse Adoptium, runtime: /opt/java/openjdk

[2022-02-08T03:41:05.381Z] Default locale: en_US, platform encoding: UTF-8

[2022-02-08T03:41:05.381Z] OS name: "linux", version: "5.4.172-90.336.amzn2.x86_64", arch: "amd64", family: "unix"
```

Reply:
hehe good catch I was blindly trusted the green

Reply:
> Java version: 17.0.2, vendor: Eclipse Adoptium

lgtm